ui/src/dataLoaders/actions/dataLoaders.ts

```diff
@@ -33,6 +33,7 @@ import {
   BundleName,
   ConfigurationState,
 } from 'src/types/dataLoaders'
+import {ILabel} from '@influxdata/influx'
```

Comment:
What about this one?

Reply:
@glinton was in the process of tossing comments on the PR:

https://github.com/influxdata/influxdb/pull/16391#discussion_r362660986